src/com/vivian/java/leetcode/Q125_ValidPalindrome.java

```diff
@@ -8,5 +8,41 @@
  * is not a palindrome.
  */
 public class Q125_ValidPalindrome {
+    public boolean ValidPalindrome(String str) {
+        if (str.isEmpty() || str == null) {
+            return false;
```

Comment:
If this were an interview, you should clarify what's the desired result for null or empty string. 

Use comment to clarify your assumption. (Your assumption might be very easy to guess, however, a comment indicates you realize that this is a clarify question you should ask in an interview)

For example: 
// Assuming false is desired when input string is null or empty